tests/everflow/test_everflow_testbed.py

```diff
@@ -74,7 +74,7 @@ class EverflowIPv4Tests(BaseEverflowTest):
 
     DEFAULT_SRC_IP = "20.0.0.1"
     DEFAULT_DST_IP = "30.0.0.1"
-    MIRROR_POLICER_UNSUPPORTED_ASIC_LIST = ["th3"]
+    MIRROR_POLICER_UNSUPPORTED_ASIC_LIST = ["th3", "j2c+"]
```

Comment:
can you 'jr2' as well?

Reply:
Added "jr2" to MIRROR_POLICER_UNSUPPORTED_ASIC_LIST